tree_unique_args/src/loop_invariant.egg

```diff
@@ -0,0 +1,43 @@
+;; Loop Invariant
+(relation find-inv-Expr (Expr Expr))
```

Comment:
Could this be a more generic "loop-child" relation that can be re-used by other optimizations?

Reply:
I think the generic one is is-inv-Expr function